common_operations/create_index.json

```diff
@@ -0,0 +1,6 @@
+{
+    "operation": {
+        "operation-type": "create-index",
+        "settings": {{index_settings | default(default_index_settings | default({})) | tojson}}
```

Comment:
Just curious, is `default({}))` necessary if the workload has nothing specified?
```
{% with default_index_settings={} %}
```

Reply:
I found that when I removed that final default({}) I was getting jinja2 templating errors, even if I specified default_settings={} in the geonames workload

Reply:
It's necessary because when `default_index_settings` is not specified,  the `{}` is needed for the JSON to be valid.